src/nvim/move.PVS-Studio.cfg

```diff
@@ -0,0 +1,10 @@
+
+source-file=/home/zyx/a.a/Proj/c/neovim/src/nvim/move.c
```

Comment:
Have no idea how it appeared, I definitely did not create it when playing with PVS-Studio.